src/transformer/events/core/badge_revoked.php

```diff
@@ -41,7 +41,6 @@ function badge_revoked(array $config, \stdClass $event) {
     $revoker = utils\get_user($config, $repo->read_record_by_id('user', $event->userid));
     $course = $badge->courseid ? $repo->read_record_by_id('course', $badge->courseid) : null;
     $lang = $badge->language ?? 'en';
```

Comment:
maybe fall back to the course lang after the badge lang, then after that config source lang

Reply:
this is still not updated

Reply:
changed to be consistent